azure-pipelines.yml

```diff
@@ -76,6 +76,7 @@ jobs:
           coverage xml
           codecov --file ./coverage.xml --token 4020cb2c-5787-485a-92f2-5d106dff85a1
           coverage html
+        condition: and( eq( variables['Agent.OS'], 'Linux' ), eq( variables['python.version'], '3.7' ) )
         displayName: Upload Coverage Report
       
       - task: PublishCodeCoverageResults@1
```

Comment:
You can probably add the conditional statement to the "Publish Coverage Results" task too so that we don't get a codecov warning

![image](https://user-images.githubusercontent.com/33552857/75999755-5160f400-5ec8-11ea-8a54-5ca3686059f5.png)